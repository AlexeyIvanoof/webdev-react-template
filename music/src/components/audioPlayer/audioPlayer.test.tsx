import ReduxProvider from "@/store/ReduxProvider";
import AudioPlayer from "./AudioPlayer";
import { render } from "@testing-library/react";
 
 describe("AudioPlayer component", () => {
    it("renders correctly", () => {
      const { container } = render(
      <ReduxProvider>
        
          <AudioPlayer />
        
      </ReduxProvider>);
      expect(container).toMatchSnapshot();
    });
  }); 