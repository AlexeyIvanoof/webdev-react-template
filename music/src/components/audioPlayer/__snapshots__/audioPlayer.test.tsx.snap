// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer component renders correctly 1`] = `<div />`;
